codegen/type_converter.go

```diff
@@ -638,25 +689,61 @@ func (c *TypeConverter) genStructConverter(
 			var (
 				stFromPrefix = keyPrefix
 				stFromType   compile.TypeSpec
+				fromTypeName string
 			)
 			if fromField != nil {
 				stFromType = fromField.Type
 				stFromPrefix = keyPrefix + pascalCase(fromField.Name)
+
+				var err error
+				fromTypeName, err = c.getIdentifierName(stFromType)
```

Comment:
if you're not handling the error, this does the same:
```
//var err error
fromTypeName, _ = c.getIdentifierName(stFromType)
// if err != {
// fromTypeName = ""
// }
```
and why don't we handle the error here?

Reply:
There's ```c.getIdentifierName(toFieldType)``` and ```c.getIdentifierName(stFromType)```. For toFieldType, I added a call upstream, with normal error checking, so the downstream one should be fine. For stFromType, it wasn't there before and in some cases it fails (when fieldMap is used a certain way), which is fine because I only need it to generate the function wrappers, and in cases when fieldMap is used in that way, no wrapper is necessary (I just don't generate a wrapper when it fails)